Scripts/Core/EditorVR.Rays.cs

```diff
@@ -22,6 +22,10 @@ partial class EditorVR
 		DefaultProxyRay m_ProxyRayPrefab;
 
 		readonly Dictionary<Transform, DefaultProxyRay> m_DefaultRays = new Dictionary<Transform, DefaultProxyRay>();
+		public List<IProxy> proxies
```

Comment:
EditorVR is an internal class, so we're not looking to expose it as a singleton. Would it be possible for you to collect all proxies via FindObjectsOfType?